documentation/reference/feature-flags.md

```diff
@@ -0,0 +1,21 @@
+# Feature Flags Supported by Charmed Superset
```

Comment:
```suggestion
# Supported Feature Flags 
```
We can drop Charmed Superset from titles in general, the user already assumes they are looking at Charmed Superset docs

Reply:
Btw, do you use an acronym for Charmed Superset?

Reply:
Makes sense. And no, no acronym at the moment (could maybe use `SS`). We often refer to it as just Superset but this doesn't really distinguish between upstream superset and our charmed version.